constrain/library/G36CoolingOnlyTerminalBoxHeatingAirflowSetpoint.py

```diff
@@ -1,30 +1,43 @@
 """
-G36 2021
 ### Description
 
 Section 5.5.5.3
-
 - When the Zone State is heating, the Heating Loop output shall be mapped to the active airflow setpoint from the minimum endpoint to the heating maximum endpoint.
 
-Verification Item:
+### Code requirement
+
+- Code Name: ASHRAE Guideline 36
+- Code Year: 2021
+- Code Section: 5.5.5 Terminal Box Airflow Control
+- Code Subsection: 5.5.5.3 Heating Airflow Control
+
+### Verification Approach
+
+The verification checks that when the zone is in heating mode, the active airflow setpoint stays within appropriate boundaries based on the current operation mode. The boundaries vary depending on whether the system is in occupied, cooldown/setup/unoccupied, or warmup/setback mode.
```

Comment:
same as earlier, please check the description here.

Reply:
To me, the current description makes sense. Please let me know if you have specific language you'd like to include/fix the description.